website/blog/2019-07-03-7.5.0.md

```diff
@@ -0,0 +1,261 @@
+
+```javascript
+let newScore = person.score
+  |> double
+  |> add(7, ?)
+  |> boundScore(0, 100, ?);
+```
+
+Note that, while it may look the same as the "Smart" pipeline variant, the partial application proposal only supports `?` in function call parameters. This means that, for example, `person |> #.score` is a valid "Smart" pipeline whose F# equivalent must use an arrow function: `person |> p => p.score`.
+
+The F# pipeline operator is also different regarding how `await` is handled:
+
+<table class="danger-force-code-no-wrap">
+  <thead>
+    <tr>
+      <th>Current JavaScript</th>
+      <th>F# pipeline</th>
+      <th>Smart pipeline</th>
+    </tr>
+  </thead>
+  <tr>
+
+<!-- DO NOT indent this table, or the code won't be highlighted -->
+<td>
+
+```js
+let id = (
+  await (
+    await fetch(url)     
+  ).json()
+).ID;
+```
+</td>
+
+<td>
+
+```js
+let newScore = fetch(url)
+  |> await
+  |> r => r.json()
+  |> await
+  |> obj => obj.ID;
+```
+</td>
+
+<td>
+
+```js
+let newScore = fetch(url)
+  |> await #
+  |> #.json()
+  |> await #
+  |> #.ID;
+```
+</td>
+
+</tr>
+</table>
+
+If you want to test this new proposal variant, you can add `@babel/plugin-proposal-pipeline-operator` to your Babel configuration:
+
+```javascript
+module.exports = {
+  plugins: [
+    ["@babel/proposal-pipeline-operator", { proposal: "fsharp" }]
+  ]
+};
+```
+
+You can also try it out in the [repl](https://babeljs.io/repl/version/%5E7.5.0#?presets=stage-1), by enabling the "Stage 1" preset.
+
+### Dynamic import ([#9552](https://github.com/babel/babel/pull/9552) and [#10109](https://github.com/babel/babel/pull/10109))
+
+Although Babel has had support for parsing dynamic imports `import(source)` for a long time, there hasn't been a consistent way of transforming it.
+- If you used `webpack` or `rollup`, you would only include `@babel/plugin-syntax-dynamic-import` and not transform it with Babel
+- If you used Node, you could use the [`babel-plugin-dynamic-import-node`](https://github.com/airbnb/babel-plugin-dynamic-import-node/) plugin to transform it
+- If you used SystemJS, `@babel/plugin-transform-modules-systemjs` in conjunction with `@babel/plugin-syntax-dynamic-import` transformed it without explicitly enabling a transform plugin for `import()`
+
+Since the dynamic import proposal has been recently merged into the main specification, we have decided to unify these use-cases under a single entry point: `@babel/plugin-proposal-dynamic-import`. This plugin must be used alongside one of the module transform plugins because Babel needs to know which module loading system you are targeting. When targeting CommonJS, it internally delegates to [`babel-plugin-dynamic-import-node`](https://github.com/airbnb/babel-plugin-dynamic-import-node/).
+
+For example, this is a valid configuration:
+
+```javascript
+module.exports = {
```

Comment:
Maybe not true but I wonder if just doing a diff would be helpful

```diff
+ 
-
```

type thing